client/src/Component/MainTable.js

```diff
@@ -70,6 +74,7 @@ const MainTable = () => {
 					{clientData.map((client) => {
 						return (
 							<tr key={client.client_id}>
+								<td><Checkbox isChecked={props.checkedCheckboxes.find((id)=>id===client.client_id)!== undefined} onChange={(checked) => props.onHandleSelectedUserState(checked, client.client_id)} /></td>
```

Comment:
Instead of `find`, let's use `includes` which is closer to what we're wanting to do here